xds/internal/balancer/cdsbalancer/cdsbalancer_test.go

```diff
@@ -795,3 +843,136 @@ func (s) TestParseConfig(t *testing.T) {
 		})
 	}
 }
+
+func (s) TestOutlierDetectionToConfig(t *testing.T) {
+	oldOutlierDetection := envconfig.XDSOutlierDetection
+	envconfig.XDSOutlierDetection = true
+	defer func() {
+		envconfig.XDSOutlierDetection = oldOutlierDetection
+	}()
```

Comment:
You don't need to set env var for this test, right?

Reply:
Done.